.travis.yml

```diff
@@ -17,67 +17,17 @@ matrix:
       script:
           # bionic has lxd from deb installed, remove it first to avoid
           # confusion over versions
+          - echo $TRAVIS_BRANCH
+          - echo $TRAVIS_BUILD_DIR
+          - cd $TRAVIS_BUILD_DIR/..
+          - tar -zcvf pr_source.tar.gz ubuntu-advantage-client
```

Comment:
I'd like to do this work only if BUILD_PR is provided. so maybe we do it in python
we could do the following directly in features.util:lxc_push_source_pull_deb_pkg
```
if not os.path.exists('../pr_source.tar.gz'):
    orig_cwd = os.getcwd()
    os.chdir('..')
    util.subp(['tar', '-zxvf', 'pr_source.tar.gz', 'ubuntu-advantage-client'])
    os.chdir(orig_cwd)
```

Reply:
Ok, thanks but you are extracting it (zXvf) and this code is creating (-c) the tar.gz. 
I think your point it to only create the tar.gz when BUILD_PR=1 (as the next task will be - if it is a cron job, so BUILD_PR=0 and then run it from daily PPA and not from the source code.

I need to create the file via travis because after we run `make test` it is going to run inside tox virtualenv, so we'd need to send the $TRAVIS_BUILD_DIR to python/behave,  which I believe is not necessary. 
I think the new modification that I've made address what you'd like to have.
I'm adding a new condition. What do you think?